src/voice_public.js

```diff
@@ -245,16 +245,14 @@ go.app = function() {
                 });
         });
 
-        // FreeText st-01
+        // EndState st-01
         self.add('state_already_registered_baby', function(name) {
             var speech_option = 1;
-            return new FreeText(name, {
-                question: $('You are already subscribed. To go back to main menu, 0 then #'),
+            return new EndState(name, {
+                text: $('You are already subscribed. To go back to main menu, 0 then #'),
```

Comment:
Can you clean up this text to be just `You are already subscribed` - it's never seen but might be confusing to a future coder